### PR TITLE
Example fix.

### PR DESCRIPTION
**Pull request by NoiSek**
Not meant to be merged. See: https://github.com/trueadm/babel-plugin-inferno/issues/6

### CODE REVIEW

File: lib/helpers/to-reference.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-"use strict";
```
Comment (NoiSek) [inline lib/helpers/to-reference.js]: `"use strict"` caused the `cons`t below to fail.

File: lib/index.js [hunk 2]
```diff
@@ -187,7 +187,7 @@ module.exports = function(options) {
 	return {
 		visitor : {
 			JSXElement: {
-				enter(path, scope) {
```
Comment (NoiSek) [inline lib/index.js]: In all the examples I've seen, this has worked, but the parent file has been an ES6 one. Seems to work just fine once you change it from an inline function into a normal `key: value` though.

**Comment by NoiSek** [comment]
I assumed adding a .babelrc to the root of the project with the `es2015` preset would solve the problem, but it does not. It doesn't appear to have any effect all actually. Babel seems to ignore local .babelrc configs in favor of the highest in the chain (Even if the config values are non-overlapping, non-existent, or a root level .babelrc is not even present).

**Comment by trueadm** [comment]
Is this still an issue @NoiSek?

**Comment by NoiSek** [comment]
No, appreciate the help. 

3 beers to @trueadm, @ChangeTip

**Comment by changetip** [comment]
Hi @trueadm, @NoiSek sent you a tip worth 3 beers ($10.50), and I'm here to deliver it ➔ **[collect your tip](https://www.changetip.com/collect/941529)**.



**[Learn more about ChangeTip](https://www.changetip.com/tip-online/github)**

**Pull request closed without merging**